src/relay_pic/mcc_generated_files/device_config.h

```diff
@@ -0,0 +1,55 @@
+/**
+  @Generated PIC10 / PIC12 / PIC16 / PIC18 MCUs Header File
+
+  @Company:
+    Microchip Technology Inc.
+
+  @File Name:
+    mcc.c
+
+  @Summary:
+    This is the device_config.h file generated using PIC10 / PIC12 / PIC16 / PIC18 MCUs
+
+  @Description:
+    This header file provides implementations for driver APIs for all modules selected in the GUI.
+    Generation Information :
+        Product Revision  :  PIC10 / PIC12 / PIC16 / PIC18 MCUs - 1.78.1
+        Device            :  PIC16F1826
+        Driver Version    :  2.00
+    The generated drivers are tested against the following:
+        Compiler          :  XC8 2.10 and above or later
+        MPLAB             :  MPLAB X 5.30
+*/
+
+/*
+    (c) 2018 Microchip Technology Inc. and its subsidiaries. 
+    
+    Subject to your compliance with these terms, you may use Microchip software and any 
+    derivatives exclusively with Microchip products. It is your responsibility to comply with third party 
+    license terms applicable to your use of third party software (including open source software) that 
+    may accompany Microchip software.
+    
+    THIS SOFTWARE IS SUPPLIED BY MICROCHIP "AS IS". NO WARRANTIES, WHETHER 
+    EXPRESS, IMPLIED OR STATUTORY, APPLY TO THIS SOFTWARE, INCLUDING ANY 
+    IMPLIED WARRANTIES OF NON-INFRINGEMENT, MERCHANTABILITY, AND FITNESS 
+    FOR A PARTICULAR PURPOSE.
+    
+    IN NO EVENT WILL MICROCHIP BE LIABLE FOR ANY INDIRECT, SPECIAL, PUNITIVE, 
+    INCIDENTAL OR CONSEQUENTIAL LOSS, DAMAGE, COST OR EXPENSE OF ANY KIND 
+    WHATSOEVER RELATED TO THE SOFTWARE, HOWEVER CAUSED, EVEN IF MICROCHIP 
+    HAS BEEN ADVISED OF THE POSSIBILITY OR THE DAMAGES ARE FORESEEABLE. TO 
+    THE FULLEST EXTENT ALLOWED BY LAW, MICROCHIP'S TOTAL LIABILITY ON ALL 
+    CLAIMS IN ANY WAY RELATED TO THIS SOFTWARE WILL NOT EXCEED THE AMOUNT 
+    OF FEES, IF ANY, THAT YOU HAVE PAID DIRECTLY TO MICROCHIP FOR THIS 
+    SOFTWARE.
+*/
+
+#ifndef DEVICE_CONFIG_H
+#define	DEVICE_CONFIG_H
+
+#define _XTAL_FREQ 500000

```

Comment:
I'd bump this up to at least 1 MHz. 500 kHz is very slow. For comparison, the rocketcan boards run at 12 MHz and your average run of the mill arduino uno runs at 16 MHz.

Note that this macro doesn't change the crystal speed itself, so you need to change the configuration and make this macro match it.